FiberPho_Main/FiberClass.py

```diff
@@ -1,3 +1,4 @@
+#junk so i can make this a pr
 from os import error
```

Comment:
These should be sorted. Theres a million tools to do it automatically, if you're working in vscode it will get done automagically